packages/dev/serializers/src/glTF/2.0/Extensions/KHR_texture_transform.ts

```diff
@@ -83,14 +83,15 @@ export class KHR_texture_transform implements IGLTFExporterExtensionV2 {
                 return;
             }
 
+            if (babylonTexture.uAng !== 0 || babylonTexture.vAng !== 0) {
+                Tools.Warn(`${context}: Texture ${babylonTexture.name} with rotation in the u or v axis is not supported in glTF.`);
+                resolve(null);
+            } else 

```

Comment:
nit: might want to collapse this with the `if` as is the typical style

`else if (`